packages/typegpu/src/core/function/tgpuFragmentFn.ts

```diff
@@ -84,9 +90,11 @@ function createFragmentFn(
   type This = TgpuFragmentFn;
 
   const core = createFnCore(shell, implementation);
+  const Output = createOutputStruct(core, implementation, shell.returnType);
```

Comment:
I did it initially because it's a schema, and we write schemas in PascalCase, but we can debate on what we should do here. An alternate argument is that other resources have schemas on them as well, but they are written in camelCase, like `dataType` for buffers or `returnType` for functions.

Reply:
So maybe `outputType` instead?

Reply:
I like it 👍